include/openssl/rsa.h

```diff
@@ -311,7 +311,6 @@ struct rsa_st {
 #define RSA_R_DATA_TOO_SMALL_FOR_KEY_SIZE 117
 #define RSA_R_DIGEST_TOO_BIG_FOR_RSA_KEY 118
 #define RSA_R_D_E_NOT_CONGRUENT_TO_1 119
-#define RSA_R_EMPTY_PUBLIC_KEY 120
```

Comment:
I suggest we just drop this commit. These error codes aren't used in *ring*. instead, they exist only to make it easier to upstream code to BoringSSL. (Technically, they could all be removed in *ring* because `OPENSSL_PUT_ERROR` is a no-op in *ring*.)